### PR TITLE
 log VM power state if there is state mismatch

### PR DESCRIPTION
**Pull request by emlin**
help #6127 to log out more message

### CODE REVIEW

File: lib/portlayer/exec/commit.go [hunk 1]
```diff
@@ -138,6 +138,11 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 			if s.ExtraConfig != nil && h.TargetState() == StateStopped && h.Runtime.PowerState != types.VirtualMachinePowerStatePoweredOff {
 				detail := fmt.Sprintf("Reconfigure: collision of concurrent operations - expected power state poweredOff, found %s", h.Runtime.PowerState)
 				log.Warn(detail)
+
+				// log out current vm power state and runtime power state got from refresh, to see if there is anything mismatch,
+				// cause in issue #6127, we see the runtime power state is not updated even after 1 minute
+				ps, _ := h.vm.PowerState(ctx)
+				log.Debugf("Container %s power state: %s, runtime power state", h.ExecConfig.ID, ps, h.Runtime.PowerState)
```
Comment (caglar10ur) [inline lib/portlayer/exec/commit.go]: you have 3 arguments but 2 formatting verbs here

File: lib/portlayer/exec/commit.go [hunk 2]
```diff
@@ -138,6 +138,11 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 			if s.ExtraConfig != nil && h.TargetState() == StateStopped && h.Runtime.PowerState != types.VirtualMachinePowerStatePoweredOff {
 				detail := fmt.Sprintf("Reconfigure: collision of concurrent operations - expected power state poweredOff, found %s", h.Runtime.PowerState)
```
Comment (caglar10ur) [inline lib/portlayer/exec/commit.go]: Ad @dougm suggested we should add %T here

File: lib/portlayer/exec/commit.go (hunk 1 above)
Comment (emlin) [reply lib/portlayer/exec/commit.go]: oh.. no

File: lib/portlayer/exec/commit.go (hunk 2 above)
Comment (caglar10ur) [reply lib/portlayer/exec/commit.go]: Nevermind this, I mixed up files :)

**Pull request merged**